esp-config/src/generate/mod.rs

```diff
@@ -51,6 +52,10 @@ impl fmt::Display for Error {
 /// that match the given prefix. It will then attempt to parse the [`Value`] and
 /// run any validators which have been specified.
 ///
+/// [`Stability::Unstable`] features will only be enabled if the `unstable`
+/// feature is enabled in the dependant crate. If the `unstable` feature is not
```

Comment:
Is a build error a good idea? I'm not sure silently ignoring is a good idea either, but I'm worried in some complex project setups this might cause issues :thinking:

Reply:
Just like unstable code isn't available unless enabled, unstable config shouldn't be, either. Complex projects will use a build script hopefully, so they'll be able to work with this. Or they'll be forced to work with this, I guess, but I see that as a benefit.